backend/de.metas.async/src/main/sql/postgresql/system/80-async-dlm/5624381_sys_gh12591_migrate_c_queue_package_processor_ID.sql

```diff
@@ -0,0 +1,17 @@
+update c_queue_workpackage
+set c_queue_packageprocessor_id = block.c_queue_packageprocessor_id, updatedBy=99, updated=TO_TIMESTAMP('2022-02-04 12:16:30', 'YYYY-MM-DD HH24:MI:SS')
+from c_queue_block block
+where block.c_queue_block_id = c_queue_workpackage.c_queue_block_id
+;
+
+alter table dlm.c_queue_workpackage_archived ADD c_queue_packageprocessor_id NUMERIC(10) DEFAULT NULL
+;
+
+update dlm.c_queue_workpackage_archived
+set c_queue_packageprocessor_id = block.c_queue_packageprocessor_id, updatedBy=99, updated=TO_TIMESTAMP('2022-02-04 12:16:30', 'YYYY-MM-DD HH24:MI:SS')
+from c_queue_block block
+where block.c_queue_block_id = dlm.c_queue_workpackage_archived.c_queue_block_id
+;
+
+alter table dlm.c_queue_workpackage_archived alter column c_queue_packageprocessor_id set not null
```

Comment:
I think DDL and DML operations on the same table won't work if both are in the same migration script.
I suggest moving that line to a new script.